cloudbuild.json

```diff
@@ -0,0 +1,253 @@
+{
+  "steps": [
+    {
+      "id": "gcr.io/blueshift-playground/cloudbuild_test:gpu-base-100",
```

Comment:
it'd be nice to explicitly say `cuda100` vs `100`, wdyt?

Reply:
works for me, I'll change that